mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/97 - network UI

Closes #97 

To test:
- Play introductory server and buy computers for two users
- Click Network toggle to open network screen
- Click workstations (the ones with _2 and _3 at the end of their names)
- Click on LAN 1 to connect computer to LAN 1 network
- Make sure objective completes when both computers are connected to network
